docs/xdump.md

```diff
@@ -754,6 +762,21 @@ The default setting of the `request` suboption for Java dump files is `request=e
 -Xdump:java:request=exclusive
 ```
 
+The Java dump agent ignores the `request=exclusive` setting if a `user` event occurs and another event already has exclusive access. In this scenario, the Java dump agent shares the access instead. This behavior is useful because it allows you to obtain a Java dump file during a deadlock situation, when exclusive access is not released. However, the resulting Java dump file, even in other situations, might omit thread stacks and contain inconsistent thread information, as indicated by the following line in the file:
+
+```
+1TIPREPINFO    Exclusive VM access not taken: data may not be consistent across javacore sections
+```
+
+On operating systems other than Windows, you can enforce exclusive access and obtain a complete dump file by specifying that the `user2` event triggers the Java dump agent instead of the `user` event. For example:
+
+```
+-Xdump:java:events=user2,request=exclusive+prepwalk
+```
+When a `user2` event occurs, for example, when you enter `kill -USR2 $PID` on the command line, the Java dump agent accepts the `request=exclusive` setting and waits for exclusive access before creating the Java dump file.
```

Comment:
I don't think `$PID` should be used, this wouldn't work. Instead `<pid>` should be used, which means the user must fill in the pid of the process that will receive the signal.